pkg/kubectl/describe.go

```diff
@@ -173,13 +173,24 @@ func (d *NamespaceDescriber) Describe(namespace, name string, describerSettings
 	}
 	resourceQuotaList, err := d.Core().ResourceQuotas(name).List(api.ListOptions{})
 	if err != nil {
-		return "", err
+		if errors.IsNotFound(err) {
+			// Server does not support resource quotas.
+			// Not an error, will not show resource quotas information.
+			resourceQuotaList = nil
+		} else {
+			return "", err
```

Comment:
This is fine and mimics what we do in a few other places.  Please open a follow up issue for 1.5 though - we have other cases that probably need to be unified here.